onnxruntime/core/providers/acl/nn/conv.cc

```diff
@@ -207,10 +207,10 @@ Status Conv<T>::Compute(OpKernelContext* context) const {
 #endif
       if(optimizable) {
         //optimized depthwise convolution
-#if defined(ACL_1902) || defined(ACL_1905)
-        auto layer = std::make_shared<arm_compute::NEDepthwiseConvolutionLayer>();
+#ifdef ACL_1902
```

Comment:
I think it needs to be reverted back exactly as it was before,
where NEDepthwiseConvolutionLayer3x3 is used for ACL_1902 and ACL_1905
and NEDepthwiseConvolutionLayerOptimized is uesd for ACL_1908

reference:
ACL_1902
https://github.com/ARM-software/ComputeLibrary/blob/v19.02/arm_compute/runtime/NEON/functions/NEDepthwiseConvolutionLayer.h#L56

ACL_1905
https://github.com/ARM-software/ComputeLibrary/blob/v19.05/arm_compute/runtime/NEON/functions/NEDepthwiseConvolutionLayer.h#L57

ACL_1908
https://github.com/ARM-software/ComputeLibrary/blob/v19.08/arm_compute/runtime/NEON/functions/NEDepthwiseConvolutionLayer.h#L81

Reply:
NEDepthwiseConvolutionLayer3x3Kernel.h  is included for ACL_1902(https://github.com/microsoft/onnxruntime/blob/5c7f6164310ffcf0a6021195ecae1e4a1654ecd9/onnxruntime/core/providers/acl/nn/conv.cc#L28).
The previous code does not work. It gives error.

Reply:
> NEDepthwiseConvolutionLayer3x3Kernel.h is included for ACL_1902(
> 
> https://github.com/microsoft/onnxruntime/blob/5c7f6164310ffcf0a6021195ecae1e4a1654ecd9/onnxruntime/core/providers/acl/nn/conv.cc#L28
> 
> ).
> The previous code does not work. It gives error.

but that's the kernel, not the function. 
I'm still a little confused. 
I don't think your change will work either, given that NEDepthwiseConvolutionLayerOptimized doesn't exist in ACL_1905 in 
arm_compute/runtime/NEON/functions/NEDepthwiseConvolutionLayer.h
Maybe multiple changes are needed.
@mikecaraman , can you chime in?

Reply:
I have built onnxruntime with the current changes successfully on Jetson Nano.

Reply:
> I have built onnxruntime with the current changes successfully on Jetson Nano.

against which ACL version?
note that by default. if you pass /.build.sh --use_acl  , it is defining ACL_1905 by default

see this sequence:
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/build.py#L300
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/build.py#L567
https://github.com/microsoft/onnxruntime/blob/master/cmake/CMakeLists.txt#L486

Reply:
Yes, ACL_1905 is default and that's the version this change is about. ACL_1905 does not have NEDepthwiseConvolutionLayer3x3Kernel defined.

Reply:
ACL_1905 has NEDepthwiseConvolutionLayer3x3Kernel defined:
https://arm-software.github.io/ComputeLibrary/v19.05/classarm__compute_1_1_n_e_depthwise_convolution_layer3x3.xhtml
https://arm-software.github.io/ComputeLibrary/v19.05/_n_e_depthwise_convolution_layer_8h_source.xhtml

The problem might be the ACL version used for Jetson Nano, the documentation points to the master version: 
https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#build-instructionsjetson-nano

Reply:
Yes, you are correct. I'll update the PR.